shared/getMetrics.js

```diff
@@ -262,8 +262,8 @@ const getMetrics = ({
         testsCount,
         testsFailedCount,
         unexpectedBehaviorCount,
-        highImpactPassedAssertionCount,
-        highImpactFailedAssertionCount,
```

Comment:
The name `highImpactFailedAssertionCount` is still present in `server/util/getMetrics.js` is there a plan to remove it from there?

Reply:
I think this comment may have been before the latest changes from main were merged in here, so this comment may be outdated. `server/util/getMetrics.js` isn't on this branch anymore.

Reply:
You're right!